pkgs/development/python-modules/acstore/default.nix

```
@@ -0,0 +1,21 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  pname = "acstore";
+
+  version = "20230226";
+
```

Later releases will have a constraint on [Python > 3.8](https://github.com/log2timeline/acstore/blob/20240407/setup.cfg#L25).

```suggestion

  disabled = pythonOlder "3.8";
```